core/org.eclipse.cdt.ui/src/org/eclipse/cdt/internal/ui/newui/Messages.properties

```diff
@@ -101,7 +101,8 @@ AbstractPage_6=Configuration:\u0020
 AbstractPage_7=Exclude resource from build
 AbstractPage_8=Cannot apply
 AbstractPage_9=Internal error
-AbstractPage_rebuildIndex_question=Changes made will not be reflected in the index until it is rebuilt. Do you wish to rebuild it now?
+AbstractPage_Rebuild_Index=Rebuild Index
+AbstractPage_rebuildIndex_question=Some build settings changes may affect the index. These changes won't take effect until the index until it is rebuilt. Do you wish to rebuild it now?
```

Comment:
Some changes to build settings may affect the index. These changes won't take effect until the index is rebuilt. Do you wish to rebuild it now?